### DISCUSSION TITLE
チュートリアルに関する意見を話す場所

### DISCUSSION THREAD

**Discussion by anatawa12** [anatawa12/AvatarOptimizer#209] [General]
チュートリアルに関する意見を話す場所です。ぜひこちらで意見を書いてください

**Comment by anatawa12** [comment]
https://twitter.com/Healsiohaiizo/status/1665349185133240320

> MergedMeshをAnon_Mergedとかにするとぱっと見の視認性は割とマシになりそうだなって思った

**Comment by hitsub** [comment]
チュートリアルはチュートリアルが必要な層に読んでもらう必要があるため、
いくつかのワードは、正確性よりも分かりやすいものに差しかえ・併記をしてもよいのかなと思いました。

- Skinned Mesh Renderer
長くて圧がありそうなのと、みなこれを正しく認識してるか怪しいと思っています。
割り切って「メッシュ」と呼んでしまっても良いかもです。
（MeshRenderとの呼び分けはボーンのあるメッシュ・ボーンのないメッシュなど…？）

- BlendShape
「シェイプキー」の方が馴染みがあると思うので、BelndShape(シェイプキー)のように書いてしまうのもアリかなと思いました。

- コンポーネント
コンポーネントの概念が周知のものか怪しいと思っているため、そもそも省いてしまっても通じるかなと思いました。

例えば以下の文は、
> このMerge Skinned Meshコンポーネントは、指定されたSkinned Mesh Rendererを一緒についているSkinned Mesh Rendererにマージするコンポーネントです。

以下のようにしても十分に伝わるかなと思いました。
> このMerge Skinned Meshは、指定されたSkinned Mesh Rendererを一緒についているSkinned Mesh Rendererにマージします。

**Reply by anatawa12** [reply]
提案ありがとうございます。修正してみました https://vpm.anatawa12.com/avatar-optimizer/beta/ja/docs/tutorial/basic-usage/

**Reply by hitsub** [reply]
読みやすくなったように思います！
置き換えした箇所等は注釈で捕捉されてて良いですね

**Comment by scuravr** [comment]
いくつか「基本的な使い方」を読みながら使おうとして戸惑った箇所があったので共有します。

1.表記について
「AAO Merge Skinned Mesh」が単に「Merge Skinned Mesh」になっている、「AAO Merge Skinned Mesh」が単に「Merge Skinned Mesh」など暗黙の読み替えが必要な箇所がありますが、認知に負荷がかかるためインスペクタ上の表記と完全に一致させてほしいです。

2.Anon_Mergedの設定
基本的な使い方の

>続いて、Anon_Mergedの設定をしましょう！
>Merge Skinned Meshは諸事情[4](https://vpm.anatawa12.com/avatar-optimizer/ja/docs/tutorial/basic-usage/#fn:4)により、ボーン、メッシュ、マテリアル、BlendShape、Bounds以外の設定を自動的には行いません（以下略）

の箇所についても、アバター改変の基本的な部分であるとは思うのですが、画像が1枚でも入っていた方が親切かと思いました。

3.アップロードについての解説
アップロードの方法について、最下部にテキストが少しでも入っていてほしいと感じました。
readmeにも

>これらのユーティリティはPlayモードに入るときかアバターをビルドするときに適用されます。

とありますが、チュートリアルにも明確に書かれていた方が親切かと思います。

よろしくお願いいたします。

**Reply by anatawa12** [reply]
2については設定されるのと、手動設定が必要なのを枠つけた画像を挿入すると良さそうですかね

**Comment by anatawa12** [comment]
手動Merge Skinned Meshはして問題ない場合は自動で行われるようになったので消してもいいかもしれない

**Reply by anatawa12** [reply]
https://misskey.niri.la/notes/9tfmc7jjzc

> 消して良いと思う。"よく分からない人がやるにも安全な"範囲は自動化されたことが理由としては大きい
残すとしても、basic-usageからは外して良いんじゃないかな(MergeSMRのtutorialとして分離するのは:itiri_aru:感じの)
